src/templates/fields/price.eco

```diff
@@ -13,6 +13,7 @@
     <div class='fr_item_auto'>
       <input type="text"
              data-rv-input='model.value.cents'
+             data-js-price-cents
```

Comment:
Can we just leave out this extra attribute, and select  this input using `[data-rv-input=model.value.cents]`?